test/e2e/alphaupdate/update_test.go

```diff
@@ -0,0 +1,290 @@
+/*
+Copyright 2025 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package alphaupdate
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	pluginutil "sigs.k8s.io/kubebuilder/v4/pkg/plugin/util"
+)
+
+const (
+	fromVersion = "v4.5.2"
+	toVersion   = "v4.6.0"
+)
+
+var _ = Describe("kubebuilder", func() {
+	Context("alpha update", func() {
+		var (
+			mockProjectDir  string
+			kbOldBinaryPath string
+		)
+
+		BeforeEach(func() {
+			var err error
+			By("building and installing current kubebuilder binary")
+			cmd := exec.Command("make", "install")
```

Comment:
we should not call make install? 
We have the current bin in the TestContext, we need to use it. 

https://github.com/kubernetes-sigs/kubebuilder/blob/master/test/e2e/v4/plugin_cluster_test.go#L53-L56